lfbb_cpp/inc/lfbb_impl.hpp

```diff
@@ -60,18 +65,28 @@ T *LfBb<T, size>::WriteAcquire(const size_t free_required) {
     const size_t linear_space = size - r;
     const size_t linear_free = std::min(free, linear_space);
 
-    /* Try to find enough linear space until the end of the buffer */
+    #ifdef LFBB_TRACE
```

Comment:
Please respect the formatting standard, I've added both the clang-format configuration file and a pre-commit hook which can be installed.